cinema/views.py

```diff
@@ -127,20 +133,46 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
```

Comment:
The `@extend_schema` decorator is used correctly to define the schema for the `list` action, providing detailed API documentation.